sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/CryptographyConstants.java

```diff
@@ -15,7 +17,12 @@
 
     static final String AES = "AES";
 
-    static final String AGENT_METADATA_VALUE = "JavaTrack2" + BlobCryptographyConfiguration.VERSION;
+    // This is properties file's name.
+    private static final String AZURE_STORAGE_PROPERTIES = "azure-storage-blob.properties";
```

Comment:
Does this need to be maintained as a static in this class? It is only used in one place.

Reply:
Remove the constant